ui/src/views/compute/KubernetesServiceTab.vue

```diff
@@ -403,8 +403,8 @@ export default {
           ips = ips.filter(x => x.issourcenat)
           this.publicIpAddress = ips.length > 0 ? ips[0] : null
         }
-      }).catch(error => {
-        this.$notifyError(error)
+      }).catch(() => {
+        this.publicIpAddress = null
```

Comment:
Instead of ignoring the exception, why not check if the user has permission to use the API?

Reply:
> Instead of ignoring the exception, why not check if the user has permission to use the API? 

The issue is not caused by permission. The users have permission to the api, but they cannot list public ips of a shared network which has scope=domain

Reply:
How do you know that @MejdiB is using shared networks with domain scope in their CKS clusters? Were you able to reproduce the error reported on #7681?

In any case, shouldn't we test if the user is trying to list the public IPs of a shared network with domain scope and not call the API in this case? It's better than ignoring any error that might happen on the API call.

Also, a description of the tests done would be great.

Reply:
> How do you know that @MejdiB is using shared networks with domain scope in their CKS clusters? Were you able to reproduce the error reported on #7681?

Yes, I am able to reproduce the issue in advanced zone with SG.

> 
> In any case, shouldn't we test if the user is trying to list the public IPs of a shared network with domain scope and not call the API in this case? It's better than ignoring any error that might happen on the API call.
>


this PR is the simplest way.


The better solution might be (1) listNetworks; (2) skip if network type is Shared.
but it did not work in my testing ,may require some investigation. 
> Also, a description of the tests done would be great.

Reply:
I too would like to see a test description. 
I agree that preventing a call to the server is good, but a working solution is as well ;)

I could live with a comment in the code that explains the better solution and why it was not implemented.

Reply:
test steps have been added
@JoaoJandre @DaanHoogland